2022-04-03 21:57:35,953 INFO    MainThread:15168 [wandb_setup.py:_flush():71] setting env: {}
2022-04-03 21:57:35,953 INFO    MainThread:15168 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-04-03 21:57:35,953 INFO    MainThread:15168 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-04-03 21:57:35,953 INFO    MainThread:15168 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-03 21:57:35,954 INFO    MainThread:15168 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\mahdi\Desktop\Github_Work\1\RSNA-Bone-Age\wandb\run-20220403_215735-hmugma1u\logs\debug.log
2022-04-03 21:57:35,954 INFO    MainThread:15168 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\mahdi\Desktop\Github_Work\1\RSNA-Bone-Age\wandb\run-20220403_215735-hmugma1u\logs\debug-internal.log
2022-04-03 21:57:35,954 INFO    MainThread:15168 [wandb_init.py:init():404] calling init triggers
2022-04-03 21:57:35,955 INFO    MainThread:15168 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-03 21:57:35,955 INFO    MainThread:15168 [wandb_init.py:init():460] starting backend
2022-04-03 21:57:35,955 INFO    MainThread:15168 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-04-03 21:57:35,971 INFO    MainThread:15168 [backend.py:ensure_launched():216] starting backend process...
2022-04-03 21:57:36,090 INFO    MainThread:15168 [backend.py:ensure_launched():221] started backend process with pid: 9144
2022-04-03 21:57:36,093 INFO    MainThread:15168 [wandb_init.py:init():469] backend started and connected
2022-04-03 21:57:36,095 INFO    MainThread:15168 [wandb_init.py:init():533] updated telemetry
2022-04-03 21:57:36,165 INFO    MainThread:15168 [wandb_init.py:init():563] communicating current version
2022-04-03 21:57:39,515 INFO    MainThread:15168 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-03 21:57:39,516 INFO    MainThread:15168 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-03 21:58:09,524 INFO    MainThread:15168 [interface_shared.py:_communicate_run():313] couldn't get run from backend
2022-04-03 21:58:09,524 ERROR   MainThread:15168 [wandb_init.py:init():583] backend process timed out
2022-04-03 21:58:09,524 ERROR   MainThread:15168 [wandb_init.py:init():592] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-04-03 21:58:09,543 ERROR   MainThread:15168 [wandb_init.py:init():958] error
Traceback (most recent call last):
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 931, in init
    run = wi.init()
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 596, in init
    backend.cleanup()
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\backend\backend.py", line 245, in cleanup
    self.interface.join()
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 458, in join
    super().join()
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\interface\interface.py", line 599, in join
    _ = self._communicate_shutdown()
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 455, in _communicate_shutdown
    _ = self._communicate(record)
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Python\Python39\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 218, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
